src/strategies/balance-of-subgraph/index.ts

```diff
@@ -0,0 +1,55 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-mainnet',
+  '137':
+    'https://api.thegraph.com/subgraphs/name/dinngodev/furucombo-tokenomics-polygon'
+};
+
+export const author = 'weizard';
+export const version = '0.1.0';
+
+export async function strategy(
+  _space,
+  network,
+  _provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const params = {
+    users: {
+      __args: {
+        where: {
+          id_in: addresses.map((address) => address.toLowerCase()),
+          amount_gt: 0
+        }
+      },
+      id: true,
+      amount: true
+    }
+  };
+  if (snapshot !== 'latest') {
+    // @ts-ignore
+    params.users.__args.block = { number: snapshot };
+  }
+  const result = await subgraphRequest(
+    options.subGraphURL ? options.subGraphURL : SUBGRAPH_URL[network],
```

Comment:
Thanks for you explanation, I understand now.
We create for people who need to customize their own subgraph. With this strategy, they don’t need to create a custom strategy again.  And I add 3 new params for this propose.
At this line(38), we can see that user can set their own subgraph URL in params, if they don’t set then will use the default(hardcoded) subgraph. 
If still concern about this, we can remove the hardcoded URL or replace the name with `furucombo-smart-wallet-balance`.

Reply:
@weizard Makes sense, sorry didn't notice this line before, so should be fine :) will merge this